project/AppDependencies.scala

```diff
@@ -2,26 +2,22 @@ import sbt._
 
 object AppDependencies {
 
-  private val play28Bootstrap    = "7.20.0"
-  private val playHmrcApiVersion = "7.2.0-play-28"
+  private val play30Bootstrap    = "8.5.0"
+  private val playHmrcApiVersion = "8.0.0"
 
-  private val pegdownVersion   = "1.6.0"
-  private val wireMockVersion  = "2.27.2"
-  private val mockitoVersion   = "1.16.46"
-  private val refinedVersion   = "0.9.26"
+  private val refinedVersion   = "0.11.1"
   private val scalaMockVersion = "5.1.0"
+  private val mockitoVersion   = "1.16.46"
 
   val compile = Seq(
-    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % play28Bootstrap,
-    "uk.gov.hmrc" %% "play-hmrc-api"             % playHmrcApiVersion,
+    "uk.gov.hmrc" %% "bootstrap-backend-play-30" % play30Bootstrap,
+    "uk.gov.hmrc" %% "play-hmrc-api-play-30"     % playHmrcApiVersion,
     "eu.timepit"  %% "refined"                   % refinedVersion
   )
 
   val test = Seq(
-    "org.pegdown"            % "pegdown"                 % pegdownVersion  % "test, it",
-    "com.github.tomakehurst" % "wiremock"                % wireMockVersion % "it",
+    "uk.gov.hmrc"            %% "bootstrap-test-play-28" % play30Bootstrap % "test, it",
```

Comment:
Just need to change the name here to play-30 (That's what's causing PR builder to raise the Bobby rule still)